code/style.css

```diff
@@ -1,5 +1,78 @@
 /* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
 
 body {
-  background-color: lightblue;
+  background-color: #d9d9d9;
+  margin-top: 20px;
 }
+
+h1 {
+font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+text-align: center;
+}
+
+img {
+  opacity: 0.5;
+}
+
+.box-one {
+  background-color: #f2f2f2;
+  padding: 80px;
+  margin-top: 80px;
+  margin-bottom: 10px;
+  margin-left: 500px;
+  margin-right: 500px;
```

Comment:
This could also be written in one line, like this (clockwise directions, starting with the top value, then right value etc)
`margin: 80px, 500px, 10px, 500px;`

Reply:
I tested with adding the margins on the same row, but box-one than ended expanding over the entire width of the site and not being scaled down as I thought it would. Not sure if I can do something different to make it a bit smaller

Reply:
Instead of setting fixed margins, you could try to set a fixed width and then use `margin: auto;` which is a quite useful hack to center things. Try something like this:
```
.box-one {
  background-color: #f2f2f2;
  padding: 80px;
  width: 70%;
  margin: auto;
  border-radius: 25px;
  border: 1px solid black;
}
```

Reply:
Added this code for it now and looks just like I want to, thank you!